### DISCUSSION TITLE
Flynt on Ventura with M-series Mac

### DISCUSSION THREAD

**Discussion by CodeConvergence** [flyntwp/flynt#485] [Q&A] [answered]
Hi

Is anyone using Flynt 1.4.1 on a M-series Mac running Ventura?
If so, would you mind sharing how you got the build process to work?

The first problem I encountered was with the Python version needed for node-gyp followed by node-sass (Node Sass does not yet support your current environment).
And those two probably aren't the only issues ... 
FYI, I have the following installed for this project: Node v14.21.3 - This is higher than the specification but I run v15 on my Intel machine and it was fine.
Also for reference: PHP 7.4.33, Composer 2.5.7, WordPress 6.2.2 and ACF 6.1.6.

I gave Flynt 2.0.0.a4 a quick try - changing Node to v18, but although the build ran without throwing any errors, the site had a 'critical failure' notice. Anyway, I can't use an alpha version in production, so it's not that relevant - but perhaps there is a key to my problem in the new build process...

Anyway, any insights would be much appreciated!

**Comment by timohubois** [comment]
Hey @CodeConvergence, you can try to run the commands emulated as x86: https://medium.com/swlh/run-x86-terminal-apps-like-homebrew-on-your-new-m1-mac-73bdc9b0f343

Let me know if this works?

**Comment by CodeConvergence** [comment, marked as answer]
Hi @timohubois

Thanks for your reply.

Unfortunately, this did not work. Whether setting the terminal to run in Rosetta mode or using the arch -x86_64 command the result is the same -  Error: Node Sass does not yet support your current environment: OS X Unsupported architecture (arm64) with Node.js 14.x

I tried several things including swapping out node sass for sass ()
Doing this, the errors start with: "ModuleBuildError: Module build failed (from ./node_modules/sass-loader/dist/cjs.js):
    SassError: Undefined variable."

Although Flynt v2 (alpha) uses sass rather than node sass, hacking the v1.4.0 build process to get it to work could be too difficult and time consuming considering that it also uses vite - unless you have some suggestions on this?

Changing node versions up to 16 also does not help, although Node 16 should support arm64.

Right now I'm out of ideas and so is Stack Overflow ;)

Any other thoughts on what I could try?

EDIT (10 minutes later ...):

Woohoo - Fixed!

I uninstalled node-sass again but then instead of recompiling it, I installed it as v4.14.1
And the build runs! :)

Maybe this could be useful to know for some future user.
